dame_flame/grouped_mr.py

```diff
@@ -51,18 +51,21 @@ def algo2_GroupedMR(df_all, df_unmatched, covs_match_on, all_covs, treatment_col
 
     # These are the unique values in the bi col. length = number of groups
     unique_matched_row_vals = np.unique(bi)
-        
+    
+    # This list stores all matched groups created by this function
```

Comment:
# This list stores all unique matched groups created by this function....
+ " where each matched group is represented as a list of units" or something indicating what the entries of the list are. This is nitpicky thing, I understood it this time with this comment but maybe someone less with it than me wouldn't? idk